statsmodels/base/model.py

```diff
@@ -931,6 +931,14 @@ def initialize(self, model, params, **kwd):
         if hasattr(model, 'k_constant'):
             self.k_constant = model.k_constant
 
+    @cache_readonly
+    def fittedvalues(self):
+        return self.predict()
+
+    @cache_readonly
+    def resid(self):
+        return self.model.endog - self.fittedvalues
```

Comment:
docstrings?

Reply:
description of cached attributes are still in the class docstring.
I'm not going to change that here, but I should add resid in class docstring

Reply:
> I'm not going to change that here, but I should add resid in class docstring

I'm not here to argue with the DFL, just saying readers would find it useful.